python/cli/prelude_cli/views/iam.py

```diff
@@ -31,20 +33,22 @@ def register_account(controller):
 def describe_account(controller):
     """ Get account details """
     acct = controller.get_account()
-    users = {user["handle"]: Permission(user["permission"]).name for user in acct['users']}
+    users = {user["handle"]: dict(permission=Permission(user["permission"]).name, expiries=user['expires'])
+             for user in acct['users']}
     print_json(data=dict(whoami=acct['whoami'], users=users, controls=acct['controls']))
 
 
 @iam.command('create-user')
 @click.option('-p', '--permission', help='provide a permission level', default=[p.name for p in Permission][-1],
               type=click.Choice([p.name for p in Permission], case_sensitive=False), show_default=True)
+@click.option('-d', '--days', help='days until expiration', default=365, type=int)
 @click.argument('handle')
 @click.pass_obj
 @handle_api_error
-def create_user(controller, permission, handle):
+def create_user(controller, permission, handle, days):
     """ Create a new user in your account """
-    token = controller.create_user(handle=handle, permission=Permission[permission.upper()].value)
-    click.secho(f'Created new [{permission}] user [{handle}]. Token: {token}', fg='green')
+    resp = controller.create_user(handle=handle, permission=Permission[permission.upper()].value, expires=datetime.utcnow() + timedelta(days=days))
```

Comment:
this is passing in a date time but no string formatting occurs. will this convert to JSON ok in the SDK function?